metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-profiler-env.xml

```diff
@@ -0,0 +1,167 @@
+    <value>30</value>
+    <description>If a message has not been applied to a Profile in this period of time, the Profile will be terminated and its resources will be cleaned up. This value should be defined along with profiler.ttl.units.
+      This time-to-live does not affect the persisted Profile data in HBase. It only affects the state stored in memory during the execution of the latest profile period. This state will be deleted if the time-to-live is exceeded.
+    </description>
+    <display-name>Time to Live</display-name>
+  </property>
+  <property>
+    <name>profiler_ttl_units</name>
+    <value>MINUTES</value>
+    <description>The units used to specify the profiler.ttl.</description>
+    <display-name>Time To Live Units</display-name>
+    <value-attributes>
+      <type>value-list</type>
+      <entries>
+        <entry>
+          <value>DAYS</value>
+          <label>Days</label>
+        </entry>
+        <entry>
+          <value>HOURS</value>
+          <label>Hours</label>
+        </entry>
+        <entry>
+          <value>MINUTES</value>
+          <label>Minutes</label>
+        </entry>
+        <entry>
+          <value>SECONDS</value>
+          <label>Seconds</label>
+        </entry>
+      </entries>
+      <selection-cardinality>1</selection-cardinality>
+    </value-attributes>
+
+
+  </property>
+  <property>
+    <name>profiler_hbase_table</name>
+    <value>profiler</value>
+    <description>The name of the HBase table that profile data is written to. The Profiler expects that the table exists and is writable.</description>
+    <display-name>HBase Table</display-name>
+  </property>
+  <property>
+    <name>profiler_hbase_cf</name>
+    <value>P</value>
+    <description>The column family used to store profile data in HBase.</description>
+    <display-name>HBase Table Column Family</display-name>
+  </property>
+  <property>
+    <name>profiler_hbase_batch</name>
+    <value>10</value>
+    <description>The number of puts that are written to HBase in a single batch.</description>
+    <display-name>HBase Batch Size</display-name>
+  </property>
+  <property>
+    <name>profiler_hbase_flush_interval</name>
+    <value>30</value>
+    <description>The maximum number of seconds between batch writes to HBase.</description>
+    <display-name>HBase Flush Interval</display-name>
+  </property>
+  <property>
+    <name>profiler_topology_worker_childopts</name>
+    <value/>
+    <description>Extra topology child opts for the storm topology.</description>
+    <display-name>topology.worker.childopts</display-name>
+    <value-attributes>
+      <empty-value-valid>true</empty-value-valid>
+    </value-attributes>
+  </property>
+  <property>
+    <name>profiler_topology_workers</name>
+    <value>1</value>
+    <description>The profiler storm topology workers</description>
+    <display-name>Number of Workers</display-name>
+  </property>
+  <property>
+    <name>profiler_topology_executors</name>
```

Comment:
This property is confusing to me.  It ends up being bound to the Storm topology.acker.executors property in the profiler flux file.  The indexing and enrichment topologies also have this property but it's named differently (indexing_acker_executors in the mpack and Enrichment Ackers in the Ambari UI for example).  Whatever the correct naming convention is, I think they should be consistent.

Reply:
Yeah, we named this property differently in the profiler.properties file than in indexing and enrichment and I was following the convention in the properties file.  I can adjust to make it consistent.